src/index.js

```diff
@@ -0,0 +1,34 @@
+import './style.css';
+
+const form = document.querySelector('form');
+// const input = document.querySelector('.input');
```

Comment:
Great job @kennyegun24 🤝

- [ ] You may also remove all commented code from your project as this is regarded as bad practice.